docs/explore/netlify/getting-started-with-netlify/index-getting-started-with-netlify.mdx

```diff
@@ -0,0 +1,305 @@
+---
+id: index-getting-started-with-netlify
+title: Getting Started with Netlify
+sidebar_label: Getting Started with Netlify
+slug: /explore/netlify/getting-started-with-netlify
+---
+
+import RedisCard from '@site/src/theme/RedisCard';
+
+Netlify is a popular static site hosting serverless platform. It is a popular way to build, deploy, and scale modern web applications in a much more scalable and secure way.  
+
+
+Netlify helps developers to launch websites and campaigns in minutes with no-ops. The hosting platform is built on top of JAMstack and unifies JavaScript and APIs,strong enough to create an infrastructure that’s well suited for both developers and content editors.
```

Comment:
with ~no-ops~ no fuss. ~The hosting platform is built on top of JAMstack and unifies JavaScript and APIs,strong enough to create an infrastructure that’s well suited for both developers and content editors.~ Netlify is built primarily for JAMstack sites, which unify JavaScript and APIs to allow applications that are well suited for both developers and content editors.